synapseAnnotations/data/analysis.json

```diff
@@ -395,143 +347,138 @@
         "source": "http://edamontology.org/operation_0292"
       },
       {
-        "value": "genotypeImputation", 
-        "description": "The statistical inference of unobserved genotypes", 
-        "source": ""
-      }, 
+        "value": "genotype imputation",
+        "description": "The statistical inference of unobserved genotypes.",
+        "source": "https://www.ncbi.nlm.nih.gov/pubmed/17572673"
```

Comment:
Does this method have a name or is it just one instance of "genotype imputation"?